conanfile.py

```diff
@@ -0,0 +1,77 @@
+from conans import ConanFile, CMake, tools
+
+
+class GoogleBenchmarkConan(ConanFile):
+    name = "benchmark"
+    version = "1.4.1"
```

Comment:
if this needs to be updated on release, please add it to the releasing.md document.

As an aside, this is very unfortunate. Is there no way to generate this file from cmake passing in the version as a parameter?

Reply:
The version populated from other files such as cmakefiles or headers. Here is the howto: https://docs.conan.io/en/latest/howtos/capture_version.html

Reply:
Is this version info needed always, or for releases only?
Having it always is actually actively misleading, as this is `1.4.1+git...`
Can it be present for releases only, and otherwise be `version = "unknown"` or something ?

Reply:
@LebedevRI that's a good point. maybe we can do something like have the default be empty with a check on package creation that it's been set? and then a test that we should run on releases to ensure it's set?

Reply:
Since my reply got a bit longer I answered here: https://github.com/google/benchmark/pull/647#issuecomment-408663311 😄